test/service/test_response.py

```diff
@@ -1589,6 +1589,30 @@ def test_summary_filter_none(self):
                 self.assertEqual(summary_object['labCount'], labCount)
 
 
+class TestUnpopulatedIndexResponse(WebServiceTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.index_service.create_indices(cls.catalog)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.index_service.delete_indices(cls.catalog)
+        super().tearDownClass()
+
+    def test_empty_response(self):
+        for entity_type in 'projects', 'samples', 'files', 'bundles':
```

Comment:
don't hard-code we have a list of those somewhere

Reply:
In fact, I don't see the point of repating this for every entity type. Pick one.